basic-javascript/project-1.spec.js

```diff
@@ -1,3 +1,245 @@
 const helpers = require('./project-1');
 
+//multipy by ten
+describe('multiplyByTen()', () => {
+    //low hanging fruit
+    it('should multiply provided number by ten', () => {
+        const expected = 100; 
+        const actual = helpers.multiplyByTen(10);
+        expect(actual).toEqual(expected);
+    });
+
+    //test for undefined
+    it('should return 0 when called with no value', () => {
+        const product = helpers.multiplyByTen();
+        expect(product).toBe(0);
+    });
+
+    // //string input
+    it('should throw error when called with a string', () => {
+        expect(() => {
+            helpers.multiplyByTen('five');
+        }).toThrow();
+    })
+});
+
+//subtract five
+describe('subtractFive()', () => {
+    it('should subtract five from the provided number', () => {
+        const expected = 15;
+        const actual = helpers.subtractFive(20);
+        expect(actual).toEqual(expected);
+    })
+
+});
+
+//are same length
+describe('areSameLength()', () => {
+    it('should say if the length of two inputs are equal or not', () => {
+        const expected = true;
+        const anwser = helpers.areSameLength('hi', 'yo');
+        expect(anwser).toBe(expected);
+    });
+
+});
+
+//are equal 
+describe('areEqual()', () => {
+    it('should say if two inputs are equal', () => {
+        const expected = true;
+        const answer = helpers.areEqual('hi', 'hi');
+        expect(answer).toEqual(expected);
+    });
+});
+
+//less than 90
+describe('lessThanNinety()', () => {
+    it('should say if it is less than 90', () => {
+        const expected = true;
+        const answer = helpers.lessThanNinety(75);
+        expect(answer).toEqual(expected);
```

Comment:
Also include a test where it expects false when the value is greater than or equal to 90.